### DISCUSSION TITLE
How many endpoints can be created?

### DISCUSSION THREAD

**Discussion by vacagonzalo** [OpenAMP/open-amp#470] [Q&A] [answered]
Hi, I am working with Xilinx's MPSoC Ultrascale+ and I wondering how many endpoints between the APU and RPU I can create.

I would like to know if there is a limit in the framework itself and if the target hardware could lower such limit.

I checked the documentation and different forums but I could not find the information.

Thanks in advance.

**Comment by arnopo** [comment, marked as answer]
In OpenAMP the max endpoints that can be instantiate is defined by "RPMSG_ADDR_BMP_SIZE". To check for Linux
@TanmayShah-xilinx : is limitation is valid on AMD distribution?
